siem_integrations/README.md

```diff
@@ -28,8 +28,92 @@ In order to utilize splunk2kafka, a [running Kafka instance](https://kafka.apach
 
 ## CLX Query
 
-Coming soon...
+### Overview
+CLX Query is a [Splunk](https:/www.splunk.com) application that requests CLX Query Service ([Django](https:/www.djangoproject.com) RESTful service) to execute queries using CLX python package and get results back to the Splunk GUI. CLX python package uses [BlazingSQL](https://blazingsql.com) engine internally to execute queries. [Gunicorn](https:/gunicorn.org) provides Web Server Gateway Interface to forward requests to CLX Query Service. [Supervisor](http:/supervisor.org) monitors Gunicorn Web Server to ensure it doesn't go offline.
 
+### Requirements
+Install required dependencies if not available.
+
+#### Install from Source
```

Comment:
It could be unclear if this is an "either/or" situation or an "and" situation. It makes sense to make it clear that these are two different ways to install the same dependencies and that users don't need to do both.

Also, is there a preferred way? I would assume Conda is the preferred way. If there is a preferred method, let's mark it as such.